linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_utmp/rule.yml

```
@@ -0,0 +1,45 @@
+documentation_complete: true
+
+prodtype: sle15
+
+title: 'Record Attempts to Alter Process and Session Initiation Information utmp'
+
+description: |-
+    The audit system already collects process information for all
+    users and root. If the <tt>auditd</tt> daemon is configured to use the
+    <tt>augenrules</tt> program to read audit rules during daemon startup (the
+    default), add the following lines to a file with suffix <tt>.rules</tt> in the
+    directory <tt>/etc/audit/rules.d</tt> in order to watch for attempted manual
+    edits of files involved in storing such process information:
+    <pre>-w /var/run/utmp -p wa -k session</pre>
+    If the <tt>auditd</tt> daemon is configured to use the <tt>auditctl</tt>
+    utility to read audit rules during daemon startup, add the following lines to
+    <tt>/etc/audit/audit.rules</tt> file in order to watch for attempted manual
+    edits of files involved in storing such process information:
+    <pre>-w /var/run/utmp -p wa -k session</pre>
+
+rationale: |-
+    Manual editing of these files may indicate nefarious activity, such
+    as an attacker attempting to remove evidence of an intrusion.
+
+severity: medium
+
+identifiers:
+    cce@sle15: CCE-85714-4
+
+references:
+    stigid@sle15: SLES-15-030760
+    srg@sle15: SRG-OS-000472-GPOS-00217
+    disa@sle15: CCI-000172
+    nist@sle15: AU-12(c),AU-12.1(iv)
+
+ocil_clause: 'Record attempt to alter and session information for utmp'
```

ocil_clause should be in form "the rule is not met", see suggestion:
```suggestion
ocil_clause: 'Audit rule is not present'
```

---

Fixed